content/en/docs/tasks/configure-pod-container/static-pod.md

```diff
@@ -97,17 +97,16 @@ For example, this is how to start a simple web server as a static Pod:
    EOF
    ```
 
-1. Configure your kubelet on the node to use this directory by running it with
+1. Configure the kubelet on that node to set a `staticPodPath` value in the
+   [kubelet configuration file](/docs/reference/config-api/kubelet-config.v1beta1/).  
+   See [Set Kubelet Parameters Via A Configuration File](/docs/tasks/administer-cluster/kubelet-config-file/)
+   for more information.
+
+   An alternative and deprecated method is to configure the kubelet on that node
+   to look for static Pod manifests locally, using a command line argument.
+   To use the deprecated approach, start the kubelet with the  
```

Comment:
I think this paragraph can be improved. I'm ok to let this PR merge and this can be iterated on

```suggestion
   An alternative and deprecated method is to configure the kubelet on that node
   to look for static Pod manifests locally, using a command line argument.
   To use the deprecated approach, start the kubelet with the  
```